draft-kuehlewind-quic-substrate.md

```diff
@@ -181,32 +181,31 @@ selects the proxy and specifies the requested support function. Often the server
 may not need to be aware of it, however, depending on the optimization function,
 server cooperation could be beneficial as well. However, the client and the proxy
 need a direct and secured communication channel in order to request and configure
-a service and exchange or expose the needed information and metadata. 
+a service and exchange or expose the needed information and metadata.
 
 
-## Frontend Support for Load Balancing and Migration/Mobility 
+## Frontend Support for Load Balancing and Migration/Mobility
 
-In this usage scenario the application service provider aims for flexibility in
-server selection, therefore the client communicates with a reverse proxy that may
-or may not be under the authority of the service provider. Such a proxy assists the client
-to access and select the content requested. Today reverse proxies terminate the
-connection, including the security association, and as such appear as the communication
-endpoint to the client. Terminating not only the transport connection but also the
-security association is especially problematic if the proxy provider is not under the direct
-authority of the actual service provider but a contracted third party.
+The use of proxies can provide benefits to performance, resilience and
+flexibility of access to services.
```

Comment:
I find this new first sentence a bit too generic. Maybe just talk about flexibility (of server selection?) of the access services and mobility, and not about performance and resilience? Or maybe w should rather say something about scalability...?

Reply:
I tried to fix this and removed mention of performance and resilience.